### ISSUE TITLE
Can we disable Dragend?

### ISSUE THREAD

**Issue by ianthedev** [Stereobit/dragend#38]
Instead of destroying it, can we just disable it and resume it later?

**Comment by mikehuebner** [comment]
Bumping this, this functionality would be amazing to have.

**Comment by PowrSlave** [comment]
I agree. Destroy is far too committing for situations where one needs to instantiate and destroy in a cyclical way.

**Comment by dollarich** [comment]
Well I have the same issue, please make multiple intiation possible, or tell us what to do?

**Comment by badoubadou** [comment]
You can try this : 
`disable: function() {
                var a = this.container;
                n(a, q, this._onStart);
            },`

By the way @Stereobit your non minified file (dragend.js) doesn't match your minified file (dragend.min.js), but it's a great and very useful piece of code :)

**Comment by Ian-Y** [comment]
Thanks @badoubadou I will try it if I need it again.

**Comment by joshuaiz** [comment]
Bumping this. I'm using a toggle for scroll or swipe action (both web and mobile). It works once if you toggle from scroll to swipe which initiates dragend and then back to scroll if you destroy dragend which is needed as the swipe and scroll work on the same elements.

But then if you toggle back it no longer works so disable would be a great option in this case.

Also, as an aside, the destroy method in the docs did not work for me, I used `destroy: true` as an option and that works.
